### PR TITLE
[web][api][SIMS #470]Institution User Views and edits Education Program Information details

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Enabled the editing program page.
- Fix some columns on DB that were declared as BIT and should be BOOLEAN.
- Manually fixed OpenShift DEV.

### CODE REVIEW

File: sources/packages/web/src/services/http/EducationProgramApi.ts [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
 
 export class EducationProgramApi extends HttpBaseClient {
-  public async createProgram(createProgramDto: any): Promise<void> {
+  public async getProgram(programId: number): Promise<void> {
+    try {
+      const response = await this.apiClient.get(
+        `institution/education-program/${programId}`,
+        this.addAuthHeader(),
+      );
+      return response.data;
```
Comment (ann-aot) [inline sources/packages/web/src/services/http/EducationProgramApi.ts:11]: function is having a `void` promise and data is returning

File: sources/packages/web/src/services/EducationProgramService.ts [hunk 2]
```diff
@@ -9,10 +9,21 @@ export class EducationProgramService {
     return this.instance || (this.instance = new this());
   }
 
-  public async createProgram(data: any): Promise<void> {
+  public async getProgram(programId: number): Promise<any> {
```
Comment (ann-aot) [inline sources/packages/web/src/services/EducationProgramService.ts:12]: instead of promise `any` can we create a interface

Comment (andrewsignori-aot) [reply sources/packages/web/src/services/EducationProgramService.ts:12]: Yes, I added, but right now it is not gonna pass in Sonar Cloud due to a duplication constraint.
Right now, we are trying to figure out how to configure the Sonal Cloud to ignore it.
The other way would be to share the DTOs between API and web, which we are not doing right now.
I don't like any, but it is the only way to have the PR passing the gates right now.

**Review by abschwenker — APPROVED** [review]

**Review by ann-aot — COMMENTED** [review]
LGTM 👍 Good work, added some comments

File: sources/packages/web/src/services/http/EducationProgramApi.ts (hunk 1 above)
Comment (andrewsignori-aot) [reply sources/packages/web/src/services/http/EducationProgramApi.ts:11]: Fixed to any for now until we figure out the duplication issue with Sonar Cloud or we start to share the DTOs between web/api.

**Pull request merged**